x-pack/plugins/security_solution/common/detection_engine/constants.ts

```diff
@@ -11,3 +11,17 @@ export enum RULE_PREVIEW_INVOCATION_COUNT {
   WEEK = 168,
   MONTH = 30,
 }
+
+export enum RULE_PREVIEW_INTERVAL {
+  HOUR = '5m',
+  DAY = '1h',
```

Comment:
Hey, DAY And WEEK here and in RULE_PREVIEW_FROM have the same value, does it intentional?

Reply:
Ok, it make sense!